rotkehlchen/accounting/structures/base.py

```diff
@@ -315,6 +315,24 @@ def get_identifier(self) -> str:
     def get_assets(self) -> list[Asset]:
         return [self.asset]
 
+    @classmethod
+    def resolve_subclass(cls, entry_type: HistoryBaseEntryType) -> type['HistoryBaseEntry']:
+        from rotkehlchen.accounting.structures.eth2 import (  # pylint: disable=import-outside-toplevel  # noqa: E501
+            EthBlockEvent,
+            EthDepositEvent,
+            EthWithdrawalEvent,
+        )
+        from rotkehlchen.accounting.structures.evm_event import \
```

Comment:
what's this? Where in the code have you seen us use `\` to break a line. Never do this.